packages/muelu/test/scaling/CMakeLists.txt

```diff
@@ -329,6 +329,14 @@ IF (${PACKAGE_NAME}_HAVE_TPETRA_SOLVER_STACK)
     COMM mpi # HAVE_MPI required
   )
 
+  TRIBITS_ADD_TEST(
+    Driver
+    NAME "DatabaseSchwarzSmoothing"
+    ARGS "--linAlgebra=Tpetra --xml=DatabaseSchwarzSmoo.xml --belosType=\"Fixed\ Point\" --matrix=DatabaseA.dat --rhs=DatabaseRhs.dat --tol=.1"
+    NUM_MPI_PROCS 1
```

Comment:
Same deal.

Reply:
I can definitely do that here since I have it wrapped in the `AdditiveSchwarz` solver